src/models/geograph.py

```diff
@@ -36,16 +35,26 @@
 )
 
 
+@dataclass
+class Habitat:
+    name: str
+    graph: nx.Graph
+    valid_classes: List[int]
+    max_travel_distance: float
+    add_distance: bool
+
+
 class GeoGraph:
     """Class for the fragmentation graph."""
 
     def __init__(
         self,
         data,
+        crs: str,
```

Comment:
Allow passing of custom built CRS objects in addition to epsg strings.

```suggestion
        crs: Union[str, pyproj.CRS], 
```